packages/core/src/index.ts

```diff
@@ -6,12 +6,15 @@ export * from './odata-v2';
 // The explicit export is needed to guarantee backwards compatibility, it overrides the same named functions in odata-common and odata-v4.
 export {
   CreateRequestBuilder,
-  CustomField,
   DeleteRequestBuilder,
-  Entity,
+  UpdateRequestBuilder,
   GetAllRequestBuilder,
   GetByKeyRequestBuilder,
-  UpdateRequestBuilder
+  ODataFunctionImportRequestConfig,
+  Entity,
+  CustomField,
+  entityDeserializer,
```

Comment:
I think this was never exported as the v2 deserializer but the base one. Please check this.

Reply:
Yeah I agree, but thinking about it a little bit more, with this explicit export that behaviour wouldn't change 🤔 unless I misunderstood something

Reply:
I think before this PR we have this situation:
```ts
import { entityDeserializer } from '@sap-cloud-sdk/core'; // entityDeserializer => src/odata-common/entity-deserializer
```

After this PR we have this situation:
```ts
import { entityDeserializer } from '@sap-cloud-sdk/core'; // entityDeserializer => src/odata-odata-v2/entity-deserializer
```

This is not the same as before, or did I miss anything?

Reply:
Oh true, I only saw the `EntityDeserializer` interface in odata-common and overlooked the  actual function 😅

Reply:
Alright, I've updated it.